tests/config/pytest_modules.yml

```
@@ -1797,6 +1797,10 @@ instrain/profile:
   - modules/nf-core/instrain/profile/**
   - tests/modules/nf-core/instrain/profile/**
 
+interproscan:
```

```suggestion
interproscan/profile:
```

This should fix the CI tests not running